index.js

```diff
@@ -35,7 +35,7 @@ module.exports.plugin = {
 			server.ext('onPreResponse', (request, h) => {
 				const response = request.response;
 				// check isServer cause we don't log the expected errors we return.
-				if (response.isBoom && response.isServer) {
+				if (response.isBoom && response.isServer && response.output.statusCode === 500) {
```

Comment:
This test will pass even without this change in the implementation. Is this meant to prevent logging of validation errors?

Reply:
Which test? The added?
The whole purpose of this functionality is to be able to intercept and rewrite error handling. Without this fix it would log everything that is intercepted as errors. Yes this will prevent validation errors to log as errors.

Reply:
Yes, that was a bad test, will fix

Reply:
Test fixed